src/cmd-init

```diff
@@ -116,4 +116,7 @@ mkdir -p cache
 mkdir -p builds
 mkdir -p tmp
 ostree --repo=repo init --mode=archive
-ostree --repo=repo-build init --mode=bare-user
+if ! has_privileges; then
+    LIBGUESTFS_BACKEND=direct qemu-img create -f qcow2 cache.qcow2 10G
```

Comment:
The `LIBGUESTFS_BACKEND` bit here is a no-op, `qemu-img` isn't libguestfs.  I think you meant the line below?  But we do that globally now in `libguestfs.sh` right?